common/source/docs/common-ac2_simple_geofence.rst

```diff
@@ -59,29 +59,74 @@ switches.  Like with the earlier fences, another fence is erected 20m
+
+
 Enabling the Fence in Mission Planner
 =====================================
 
-.. image:: ../../../images/Fence_MPSetup.png
-    :target: ../_images/Fence_MPSetup.png
+To enable Fences, go to the Mission Planner full parameter list (CONFIG->Full Parameter Tree), search for items with ``FENCE_``:
+
+.. image:: ../../../images/fence_enable.png
+    :target: ../_images/fence_enable.png
+
+Plane Parameter List (Copter/Rover have fewer)
 
-The Fence can be set-up by doing the following:
+-  Set :ref:`FENCE_ACTION<FENCE_ACTION>` = to whatever you wish for a breach action. These will vary depending on vehicle type.
+-  Set :ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` = to the altitude limit relative to home you want (in meters). This is unavailable in Rover.
+-  Set :ref:`FENCE_ENABLE<FENCE_ENABLE>` =1 to enable FENCEs.
+-  Set :ref:`FENCE_MARGIN<FENCE_MARGIN>` = to the distance from the fence horizontal boundary the vehicle must maintain in order to prevent a breach.
+-  Set :ref:`FENCE_RADIUS<FENCE_RADIUS>` to the maximum distance from HOME you want (in meters). This should normally be at least 50m. This value must be larger than :ref:`FENCE_MARGIN<FENCE_MARGIN>` and greater than 30m.
+-  :ref:`FENCE_TOTAL<FENCE_TOTAL>` should not be changed manually and indicates the number of inclusion/exclusion fence points stored. (See :ref:`common-polygon_fence`)
+-  Set :ref:`FENCE_TYPE<FENCE_TYPE>` = is a bitmap set to enable the various fence types: MIN or MAX altitude, simple CIRCLE tin can around HOME, or POLYGON fences. The POLYGON fences must also have been loaded via a fence list from a ground control station in order to be active.
 
--  Connect your autopilot to the Mission Planner
--  Go to the **CONFIG/GeoFence** screen
--  Enable the FENCE: :ref:`FENCE_ENABLE<FENCE_ENABLE>` =1.
+.. note:: Polygon fence type includes the circular fences specified in the Inclusion/Exclusion fence list. The simple home centered CIRCLE fence is a separate fence. Rover ignores altitudes, if set.
 
-.. note:: The fence can be enabled/disabled by an RC channel, or Mission Command (DO_FENCE_EANBLE), but must be enabled with this parameter first.
+Defaults for :ref:`FENCE_TYPE<FENCE_TYPE>` are:
 
--  Set :ref:`FENCE_TYPE<FENCE_TYPE>` = default of "ALL" is fine and will allow you to set up an additional :ref:`common-polygon_fence` , unless you want only a cylindrical fence to be active if a polygon fence has also been loaded. Then  you could select "Altitude" or "Circle" or both.
--  Set :ref:`FENCE_ACTION<FENCE_ACTION>` ="RTL or LAND" ,or whatever you wish for a breach action.
--  Set :ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` to the altitude limit you want (in meters). This is unavailable in Rover.
--  Set :ref:`FENCE_RADIUS<FENCE_RADIUS>` to the maximum distance from HOME you want (in
-   meters).  This should normally be at least 50m.
+- Rover: CIRCLE and POLYGON
```

Comment:
**_Very_** strictly speaking, Rover can indeed have minimum and maximum altitude fences set - and could have had max altitude fences enabled prior to the work I completed.

The function `AC_Fence::check()` doesn't differentiate vehicle type, so if your `FENCE_TYPE` is set to include the bits for max and min fence (you'd need to enter these manually force override the parameter present, presumably), then you could indeed have a fence violation on max/min altitude.

Realistically, this is more of an "Advanced Use Case" for Rover - I'm okay to leave this information out, just thought you may be interested to know.

Reply:
I'd argue that this is more like an undocumented feature. 

I agree that other parts of Rover may ignore altitude (eg, I can't tell a rover to loiter at an altitude because it's altitude is stuck at ground level).

However, if I'm operating on a hill side, altitude could definitely play a role. Perhaps I don't want the vehicle to be in a polygon, but do want to stop it going too far above or below my home location.

I'm not a Rover user, so I can't say I know all the standard use cases, but I don't know that I can agree it's a bug.